### PR TITLE
Update version and changelog for version 19

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

**Review by ryanpetrello — APPROVED** [review]

**Comment by shanemcd** [comment]
regate

**Review by beeankha — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,22 @@
...
+- Fix k8s credentials that use a custom ca cert (https://github.com/ansible/awx/pull/9744)
+- Fix a bug that allowed a user to attempt deleting a running job (https://github.com/ansible/awx/pull/9758)
+- Fixed the Kubernetes Pod reaper to properly delete Pods launched by Receptor (https://github.com/ansible/awx/pull/9819)
+- AWX Collection Modules: added ability to set instance groups for organization, job templates, and inventories. (https://github.com/ansible/awx/pull/9804)
+- Fixed CSP violation errors on job details and job settings views (https://github.com/ansible/awx/pull/9818)
+- Added support for convergence any/all on workflow nodes (https://github.com/ansible/awx/pull/9737)
+- Fixed race condition that causes InvalidGitRepositoryError (https://github.com/ansible/awx/pull/9754)
+- Added support for Execution Environments to the Activity Stream (https://github.com/ansible/awx/issues/9308)
+- Fixed a bug that improperly formats OpenSSH keys specified in custom Credential Types (https://github.com/ansible/awx/issues/9361)
+- Fixed an HTTP 500 error for unauthenticated users (https://github.com/ansible/awx/pull/9725)
```
Comment (jakemcdermott) [inline CHANGELOG.md:20]: - add subscription wizard: https://github.com/ansible/awx/pull/9496

Comment (jakemcdermott) [inline CHANGELOG.md:20]:
```suggestion
- Fixed an HTTP 500 error for unauthenticated users (https://github.com/ansible/awx/pull/9725)
- Added subscription wizard: https://github.com/ansible/awx/pull/9496
```

**Comment by shanemcd** [comment]
regate

**Pull request merged**